network/src/noise/handshake.rs

```diff
@@ -431,9 +477,27 @@ impl NoiseUpgrader {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, e))?;
 
         // send the response
+        sl_info!(
+            network_log(network_events::NOISE_UPGRADE, &self.network_context)
+                .data("origin", "server")
```

Comment:
Do you want to reuse the `ConnectionOrigin::Inbound/Outbound` for this log?

Reply:
well, I did, but I didn't want to pipe in too much data, I think I'll go with role as @mimoo suggested.